cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -0,0 +1,193 @@
+:_mod-docs-content-type: ASSEMBLY
+[id=“cloud-experts-rosa-hcp-activation-and-account-linking-tutorial”]
+= Tutorial: ROSA with HCP activation and account linking
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-rosa-hcp-activation-and-account-linking-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2024-02-08
+//---
+//date: '2024-02-08'
+//title: Jiri Fiala
+//authors:
+//  - Jiri Fiala
+//---
+
+This tutorial describes the process for activating {hcp-title-first} and linking to an AWS account, before deploying the first cluster.
+
+[IMPORTANT]
+====
+If you have received a private offer for the product, make sure to proceed according to the instructions provided with the private offer before following this tutorial. The private offer is designed either for a case when the product is already activated, which replaces an active subscription, or for first time activations.
+====
+
+== Subscription enablement and AWS account setup
+
+. Activate the {hcp-title} product at the AWS console page by clicking the *Get started* button:
++
+image::rosa-get-started.png[]
++
+If you have activated ROSA before but did not complete the process, you can click the button and complete the account linking as described in the following steps.
+
+. Confirm that you want your contact information to be shared with Red Hat and enable the service:
++
+image::rosa-enable-2.png[]
++
+* You will not be charged by enabling the service in this step. The connection is made for billing and metering that will take place only after you deploy your first cluster.
+* This could take a few minutes:
++
+After the process is completed, you will see a confirmation:
++
+image::rosa-prereq-enable-3.png[]
++
+Other sections on this verification page show the status of additional prerequisites. In case any of these prerequisites are not met, a respective message is shown. Here is an example of insufficient quotas in the selected region:
++
+image::rosa-service-quota-4.png[]
+
+. Click the *Increase service quotas* button or use the *Learn more* link to get more information about the about how to manage service quotas. In the case of insufficient quotas, note that quotas are region-specific. You can use the region switcher in the upper right corner of the web interface to re-run the quota check for any region you are interested in and then submit service quota increase requests as needed.
++
+If all the prerequisites are met, the page will look like this:
++
+image::rosa-prereq-5.png[]
++
+The ELB service-linked role is created for you automatically. You can click any of the small *Info* blue links to get contextual help and resources.
+
+== AWS and Red Hat account and subscription linking
+
+. Click the orange *Continue to Red Hat* button to proceed with account linking:
++
+image::rosa-continue-rh-6.png[]
+
+. If you are not already in the same browser session, you will be asked to log into your Red Hat account:
```

Comment:
s/If you are not already in the same browser session/If you are not already logged in to your Red Hat account in your current browser's session, you will be asked to log in to your account

I thought the intro clause was a little confusing, so I went for more explicit.